modules/mobile/app/models/mobile/v0/adapters/immunizations.rb

```diff
@@ -24,9 +22,10 @@ def parse(immunizations)
               manufacturer: vaccine&.manufacturer,
               note: note(immunization[:note]),
               reaction: reaction(immunization[:reaction]),
-              short_description: vaccine_code[:text]
+              short_description: immunization[:vaccine_code][:text]
             )
           end
+          vaccine_map.uniq { |immunization| [immunization[:date], immunization[:short_description]] }
```

Comment:
Does this keep the most recent duplicate?

Reply:
I guess they're the same if the date matches, does date include time? Can an entry be on the same day but the correction a couple minutes apart?

Reply:
The date does include time. No idea about the corrections. Raf's examples show the exact same time for both

Reply:
Ok, I think is the most conservative de-duping either way. I wasn't sure if the date was time received or time entered. Sounds like entered based on Raf's data.